### ISSUE TITLE
Start-HPEiLOCertificateSigningRequest Broken

### ISSUE THREAD

**Issue by rgb99** [HewlettPackard/PowerShell-ProLiant-SDK#6]
I am running into a problem with the Start-HPEiLOCertificateSigningRequest cmdlet. I pass the variables and values and the default data is returned into the certificate.

Example:
```
$connection = Connect-HPEiLO $interface -Username $striLOUsername -Password $striLOPassword -DisableCertificateAuthentication
$commonName = (Get-HPEiLOIPv4NetworkSetting -connection $connection).DNSname + "." + (Get-HPEiLOIPv4NetworkSetting -connection $connection).DomainName
Start-HPEiLOCertificateSigningRequest -Connection $connection -State "MyState" -Country "US" -City "Anytown" -Organization "MyCompany" -OrganizationalUnit "MyGroup" -CommonName "$commonName"
```
After I generate a cert file with certreq.exe and then import with Import-HPEImportCertificate, the subject values are all default except for the CN.

I also see the following in the manual for the cmdlet.

```
-IncludeiLOIP [<SwitchParameter>]
    Includes the iLO IP along with the certificate. The supported values are Yes or No. This parameter is supported only on iLO 5 servers.
```
However, this feature is absolutely available on iLO 4 as a checkbox at the Generate CSR page.

In addition, the cmdlet parameter does not recognize Yes as a value!!! So it will not work. I would love for this to get resolved as I have a scripted way of adding iLO's into a CA that is held up by this broken cmdlet.

I brought up the issue in the HPE Forums as well. https://community.hpe.com/t5/Server-Management-Remote-Server/Start-HPEiLOCertificateSigningRequest-Broken/td-p/7008357

**Comment by hp-powershell** [comment]
Hi,

I acknowledge your issue and as i said in community forum also we are working on the issue and will get back to you on the hotfix availability soon.

Thanks,
Gokul

**Comment by rgb99** [comment]
Thanks. Would love it if -IncludeiLOIP also worked with iLO v4.

**Comment by rgb99** [comment]
Fixed in 2.0.0.1
